### ISSUE TITLE
Sprayer setting says "Dispatch AC"

### ISSUE THREAD

**Issue by BilboGCL** [KnuckleCracker/CW4-bug-tracker#217]
The sprayers checkbox text says "Dispatch AC", but they dispatch bluite ore. I understand, that the text space is limited, but this might confuse players as you need the factory. 
Thinking of a map where you have a basin of AC, sending sprayers to collect the AC and using porters to send the AC/ore back is another pitfall.

**Comment by Bob2Chiv** [comment]
This particular bug can be evident on 'Shattered'.
See the picture below where I was [incorrectly] trying to use one Sprayer to feed the rest of my network
![DispatchACIssue](https://user-images.githubusercontent.com/72889843/99329487-598d7380-284c-11eb-99d6-e37f7a06f407.png)

**Comment by Karsten75** [comment]
This system is based on the same principle as in CW3 - with the exception that here, ore is dispatched to the factory and not to the Command Nodes. A sprayer cannot feed another sprayer. The ore can be collected by a porter. Last I checked, this worked correctly. 

The sprayer itself can use the surrounding AC to, for instance, fire at oncoming creeper. Again, same as in CW3 and last I verified, it worked. 

if anything, it needs a minor tweak to the description in the control panel. it might be best to expand fully in the info page on the wiki and leave the sprayer's control panel as-is.
